index.js

```diff
@@ -11,15 +11,24 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+let maxPage = 1;
+let page = 1;
 const searchQuery = "";
 
-async function fetchCharacters() {
+async function fetchCharacters(page) {
```

Comment:
You do not need to pass `page` as a parameter here. `page` is a global variable and can be accessed directly inside your function. 🙂 
```suggestion
async function fetchCharacters() {
```